bin/librancher-ecm.sh

```diff
@@ -1,19 +1,23 @@
 #!/bin/sh
 
-. libstd-ecm.sh
-
-__RANCHER_PATH="${GOPATH}/src/github.com/rancher"
 __INVALID_ARG_ERROR="error: update requires an argument"
-__IMAGE_BUILD_REPOS=$(find "${__RANCHER_PATH}" \
-    -path "*image-build-*"                     \
-    -type f -not -path "*image-build-tools/*"  \
-    -type f -not -path "*image-build-base/*"   \
-    -type f -name "Dockerfile")
 
 # rancher_list_local_repos prints out all of the local
 # repos to STDOUT.
 rancher_list_local_repos() {
```

Comment:
In the past, I've used the documented image source page as an authoritative list of repos:

```bash
REPOS=$(curl -s 'https://raw.githubusercontent.com/rancher/rke2/master/developer-docs/image_sources.md' | grep -E 'rancher/(hardened-build-base|rke2)' | awk -F '|' '{print $4}' | xargs -n1 echo | sort | uniq)
```

Reply:
I added it as the `rke2_list_docker_images` function

Reply:
The command as shown above lists the GitHub repos, not the images. If you want the images you should change `{print $4}` to `{print $3}`.

Reply:
I got confused by its output, let me update it.

Reply:
Got it. In that case, I've added both:

- `rke2_list_image_repos `
- `rke2_list_images `